test/test-shim.ts

```diff
@@ -70,6 +74,37 @@ test('modules already in cached do not show up in trace', t => {
   t.end();
 });
 
+test('preload traces from the entrypoint and writes it to an env controlled file', t => {
+  const script = join(__dirname, './fixtures/modA.js');
```

Comment:
Still uses unix paths. Do it this way: `join(__dirname, 'fixtures', 'modA.js')`. Here and elsewhere.

Reply:
`path.join` normalizes paths, so these would all work on windows. I'm happy to change them if preferred.

Reply:
Still preferable because it would be more obviously correct.